server/server.go

```diff
@@ -370,8 +372,8 @@ func cutReleaseCommandF(args []string, w http.ResponseWriter, slashCommand *MMSl
 	return nil
 }
 
-func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag, repo string) error {
-	var pluginTag = regexp.MustCompile("v[0-9]+.[0-9]+.[0-9]+$")
+func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag, repo string, force bool) error {
+	pluginTag := regexp.MustCompile("v[0-9]+.[0-9]+.[0-9]+$")
```

Comment:
Is there the start of string needed? `^` 
Should the dot be escaped?

Reply:
oo.. laser eyes @metanerd. Great catch.